sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -977,6 +977,17 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+    val OPTIMIZE_INSERT_INTO_VALUES_PARSER =
+    buildConf("spark.sql.parser.optimizeInsertIntoValuesParser")
+      .internal()
+      .doc(
+        """
+          | Controls whether we optimize the ASTree that gets generated when parsing
+          | insert into ... values DML statements.
+          |""")
```

Comment:
nit: please follow the code style and indentation of other config definitions in this file.

Reply:
Done!